polyglot.js

```diff
@@ -240,7 +244,7 @@ class PolyGlot {
     }
 
     preCreateChatMessage(data, options, userId) {
-        if (data.type == CONST.CHAT_MESSAGE_TYPES.IC) {
+        if (data.type == CONST.CHAT_MESSAGE_TYPES.IC || (this.allowOOC && data.type < 5 && data.type > 0 && game.user.isGM)) {
```

Comment:
What's type 5? and what's type 0, use the CONST value ? also why compare 0 < type < 5?
I'd prefer it to be `[CONST.CHAT_MESSAGE_TYPES.OOC, CONST.CHAT_MESSAGE_TYPES.EMOTE, CONST.CHAT_MESSAGE_TYPES.WHISPER].includes(data.type)`

Reply:
Done in latest commits